README.md

```diff
@@ -8,20 +8,139 @@ Monorepo that hosts:
 - CI/CD pipelines
 - Infrastructure
 
+## Folder Structure
+
+The folder structure of the repository is based on [Nrwl](https://nrwl.io/) [NX](https://nx.dev/angular) mono-repository project.
+
+```|
+Energinet-DataHub/greenforce-frontend
+├── apps                    # Contains source code for applications. This includes frontends, BFF and E2E. Apps are grouped by a product root folder, and type prefixed sub-folder(s).
+|  ├── dh                   # Data Hub application (product)
+|  |  ├── api-dh            # - BFF for Data Hub
+|  |  ├── app-dh            # - Frontend for Data Hub
+|  |  └── e2e-dh            # - E2E tests for Data Hub
+|  └── ett                  # Energy Track and Trace (product)
+|  |  ├── app-ett           # - Frontend for Energy Track and Trace
+|  |  └── e2e-ett           # - E2E tests for Energy Track and Trace
+├── dist                    # Contains output files when building artifacts (for distribution)
+|  ├── apps                 #
+|  └── libs                 #
+├── libs                    # Contains source code for libraries. Libraries are grouped by a product root folder, and type prefixed sub-folder(s).
+|  ├── dh                   # - Libraries for Data Hub (product)
+|  ├── ett                  # - Libraries for Energy Track and Trace (product)
+|  └── ui-watt              # - Actual implementation of the design system
+└── tools                   # Contains various tools
+   └── executors            # - Executors perform actions on your code. This can include building, linting, testing, serving. Example: "add license"
+```
+
+## App Types
+
+- api (BFF)
+- app (Frontend)
+- e2e
+
+### Api apps (BFF's)
+
+A Api app contains the BFF of a product. Api apps are located under `apps/<product>` and are prefixed with `api-<product>`.
+
+### Frontend App
+
+A Frontend app contains the frontend application of a product. Frontend apps are located under `apps/<product>` and are prefixed with `app-<product>`.
+Allowed dependencies: `shell`, `environments`, `assets`, `styles` libraries.
+
+### E2E apps
+
+A E2E app contains the e2e application of a product. Frontend apps are located under `apps/<product>` and are prefixed with `e2e-<product>`
+Allowed dependencies: `e2e-util` libraries and always an implicit dependency of an app, and a api (BFF).
+
+## Library Types
+
+All libraries are related to a product or shared across multiple products. The libraries are located under `libs/<product>` and are prefixed with `<library type>-<library name>`.
+For shared libraries `gf` should be used as `product`.
+
+No library must have any dependencies to an app. Depending on the type of a library, a library can have dependencies to other certain types of libraries.
```

Comment:
suggestion: `data-access` libraries may depend on `api` apps

Reply:
Should be fixed now.